.github/workflows/dockerhub.yml

```diff
@@ -29,16 +29,15 @@ jobs:
     # Build the Docker image
     - name: Build
       run: |        
-        docker build -t "$DOCKER_ORGANIZATION"/"$IMAGE":"$GITHUB_SHA" \
+        docker build -t "$DOCKER_ORGANIZATION"/"$IMAGE":$(echo ${GITHUB_SHA} | cut -c1-8)\
           --build-arg GITHUB_SHA="$GITHUB_SHA" \
           --build-arg GITHUB_REF="$GITHUB_REF" .
-        docker tag "$DOCKER_ORGANIZATION"/"$IMAGE":"$GITHUB_SHA" "$DOCKER_ORGANIZATION"/"$IMAGE":latest
+        docker tag "$DOCKER_ORGANIZATION"/"$IMAGE":$(echo ${GITHUB_SHA} | cut -c1-8) "$DOCKER_ORGANIZATION"/"$IMAGE":latest
 
     # Push the Docker image to Google Container Registry
     - name: Publish
       run: |
         docker login -u=$DOCKER_USERNAME -p=$DOCKER_PASSWORD 
-        docker push $DOCKER_ORGANIZATION/$IMAGE:$GITHUB_SHA
-        docker push $DOCKER_ORGANIZATION/$IMAGE:latest
+        docker push $DOCKER_ORGANIZATION/$IMAGE:$(echo ${GITHUB_SHA} | cut -c1-8)
```

Comment:
you can remove the tag here. Pushing the image without a specific tag will push both tags